boards/arm/bt510/doc/bt510.rst

```diff
@@ -0,0 +1,219 @@
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* RADIO (Bluetooth Low Energy and 802.15.4)
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UART (Universal Asynchronous Receiver-Transmitter)`
+* :abbr:`USB (Universal Serial Bus)`
+* :abbr:`WDT (Watchdog Timer)`
+
+.. figure:: img/bt510.png
+     :width: 500px
+     :align: center
+     :alt: Sentrius BT510 Sensor, front view
+
+     Sentrius BT510 Sensor, front view
+
+.. figure:: img/bt510_back.jpg
+     :width: 500px
+     :align: center
+     :alt: Sentrius BT510 Sensor, rear view
+
+     Sentrius BT510 Sensor, rear view
+
+More information about the board can be found at the
+`Sentrius BT510 website`_.
+
+Hardware
+********
+
+Supported Features
+==================
+
+The BT510 Sensor supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| USB       | on-chip    | usb                  |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See the `Sentrius BT510 website`_
+for a complete list of BT510 Sensor features.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1A (green) = P0.22
+* LED1B (red) = P0.20
+
+Push button
+------------
+
+* BUTTON = SW1 = P0.42
```

Comment:
Should be P1.10 (nRF52840 mapping) or SIO42 (BL654/schematic mapping)

Reply:
Updated.